lib/matplotlib/tests/test_ticker.py

```diff
@@ -731,6 +746,7 @@ def test_params(self, input, expected):
         Note that cases 2. and 3. are looped over several separator strings.
         """
 
+        plt.rcParams['axes.unicode_minus'] = unicode_minus
```

Comment:
Is it allowed to modify the global state in tests or should this be an `rc_context()`?

Reply:
I am not sure. Going by a previous comment by @anntzer, I assumed that the global state would be reset for each test.
> rcparams are always reset to their defaults before running the tests.

If I interpreted that incorrectly, please let me know. Thanks!

Reply:
they are always reset by https://github.com/matplotlib/matplotlib/blob/8137b4c5459c9362e704853914fbbd74ea6e3199/lib/matplotlib/testing/decorators.py#L26 which is called by https://github.com/matplotlib/matplotlib/blob/8137b4c5459c9362e704853914fbbd74ea6e3199/lib/matplotlib/testing/conftest.py#L21.